src/main/kotlin/agallipo/main.kt

```diff
@@ -1,6 +1,6 @@
 package agallipo
 
-
+//created by alex :)
 fun main(args: Array<String>) {
     val pathname = "/Users/cxb0507/Desktop/example.json"
     var boardingCardJsonParser = BoardingCardJsonParser(pathname)
```

Comment:
Great effort parsing the json, nowadays Json files are very common so probably other people have fight with them, I will try to use a library (not sure but probably JacksonMapper, Gson,...), they will save you from doing a lot of boring work.

Reply:
Thanks! I just implemented t the Gson library!